src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -482,6 +488,235 @@ describe('526 helpers', () => {
     });
   });
 });
+describe('transform', () => {
```

Comment:
I went back and forth trying to figure out if this was solving the same problem as our `schema.unit.test.js`. Where that one checks if the submission passes the schema validations, this one checks that it doesn't, for example, remove anything it shouldn't remove, which the schema validation wouldn't necessarily catch (if the property wasn't required).

Good thinking. I like it. :+1:

Might be a pain to maintain, but we'll see, eh?